src/main/java/com/minecolonies/coremod/colony/buildings/modules/AbstractCraftingBuildingModule.java

```diff
@@ -432,12 +432,17 @@ public boolean addRecipe(final IToken<?> token)
             final IRecipeStorage recipeStorage = IColonyManager.getInstance().getRecipeManager().getRecipes().get(token);
             if (recipeStorage != null)
             {
+                if (recipeStorage.getAlternateOutputs().isEmpty())
+                {
+                    //building.getColony().getRequestManager().onColonyUpdate(request -> request.getRequest() instanceof IDeliverable && ((IDeliverable) request.getRequest()).matches(recipeStorage.getPrimaryOutput()));
+                    //return true;
+                }
+
```

Comment:
Empty if?

I may be missing something, but there doesn't seem to be an actual code change here (other than some whitespace changes)
I'm wondering how this is solving the clipboard issue

Reply:
oopsy.

Reply:
The fix is actually making the ItemStackUtils a boolean and not a Boolean. But I did forgot to re-comment this =D

Reply:
I wanted to have this "quick option" as this is more performant for most things.

Reply:
Ah, I see. That part made it into another commit :smile: